.github/workflows/ci.yaml

```diff
@@ -33,8 +37,5 @@ jobs:
       - name: Lint
         # Often, lint guidelines depend on the Go version. To prevent
         # conflicting guidance, run only on the most recent supported version.
-        if: matrix.go-version == '1.22.x'
-        run: make lint
-      - name: Check Generate
-        if: matrix.go-version == '1.22.x'
-        run: make checkgenerate
```

Comment:
Why remove this check?

Reply:
It exists in the buf tests already, but I do need to make sure the buf tests run for every PR (instead of only on proto changes).

Reply:
Done in https://github.com/bufbuild/protovalidate-go/pull/139/commits/bd22c50bb5e446a9c50286701444ee21692543ff